sheetsync/sheetsync/main.py

```diff
@@ -224,6 +227,16 @@ def parse_row(self, row):
 					value = None
 					row_dict['_parse_errors'].append("Failed to parse column {}: {}".format(column, e))
 			row_dict[column] = value
+		# As a special case, add some implicit tags to the tags column.
+		# We prepend these to make it slightly more consistent for the editor,
+		# ie. it's always DAY, CATEGORY, POSTER_MOMENT, CUSTOM
+		row_dict['tags'] = (
+			[
+				row_dict['category'], # category name
+				worksheet, # sheet name
+			] + ('Poster Moment' if row_dict['poster_moment'] else [])
```

Comment:
It should be ['Poster Moment'] since you can't concat a string to a list

Reply:
whoops, thanks